seq2seq/dataset/utils.py

```diff
@@ -55,13 +56,13 @@ def prepare_data(path, src_max_len, tgt_max_len, tokenize_func=space_tokenize):
                 if filter_pair(pair, src_max_len, tgt_max_len):
                     pairs.append(pair)
             except:
-                print("Error when reading line: {0}".format(line))
+                logger.error("Error when reading line: {0}".format(line))
                 raise
             counter += 1
             if counter % 100 == 0:
-                print("\rRead {0} lines".format(counter), end="")
+                logger.info("\rRead {0} lines".format(counter))
```

Comment:
The logging prefixes are eaten by '\r', but removing '\r' will result in too many logging messages.